consensus/state.go

```diff
@@ -49,6 +49,13 @@ type State struct {
 	siafundOutputs        map[SiafundOutputID]SiafundOutput
 	delayedSiacoinOutputs map[BlockHeight]map[SiacoinOutputID]SiacoinOutput
 
+	// subscriptions is a map containing a bunch of channels that are being
+	// listend on by modules. An empty struct is thrown down the channel any
+	// time that the consensus set of the state changes. subscriptionCounter
+	// only ever increments, and prevents collisions in the map.
+	subscriptions       map[int]chan struct{}
+	subscriptionCounter int
```

Comment:
there were at some points plans to let subscriptions die/expire/timeout. If your host is restarting or something individually you'll want a way to release a subscription.

Not a big deal though. For the time being, it's identical to using a slice and calling append.